update extended config docu, fixes #515 (#525)
SAP/luigi: core/examples/luigi-sample-angular/README.md

4. Start the example application from the `luigi/core/examples/luigi-sample-angular` folder.

>    The [`src/luigi-config/extended`](src/luigi-config/extended) folder contains the default configuration including all available features. If you want to try out a simpler example, change the configuration reference in the [`index.html`](src/index.html) file to `<script src="/assets/basicConfiguration.bundle.js"></script>`. The basicConfiguration.bundly.js can be changed by editing [`src/luigi-config/basic/basicConfiguration.js`](src/luigi-config/basic/basicConfiguration.js).

Reviewer: ```suggestion
   The [`src/luigi-config/extended`](src/luigi-config/extended) folder contains the default configuration including all available features. If you want to try out a simpler example, change the configuration reference in the [`index.html`](src/index.html) file to `<script src="/assets/basicConfiguration.bundle.js"></script>`. The basicConfiguration.bundle.js can be changed by editing [`src/luigi-config/basic/basicConfiguration.js`](src/luigi-config/basic/basicConfiguration.js).
```

Reviewer: a tiny typo :)

>    The [`src/luigi-config/extended`](src/luigi-config/extended) folder contains the default configuration including all available features. If you want to try out a simpler example, change the configuration reference in the [`index.html`](src/index.html) file to `<script src="/assets/basicConfiguration.bundle.js"></script>`. The basicConfiguration.bundle.js can be changed by editing [`src/luigi-config/basic/basicConfiguration.js`](src/luigi-config/basic/basicConfiguration.js).

Reviewer 2: ```suggestion
   The [`src/luigi-config/extended`](src/luigi-config/extended) folder contains the default configuration that includes all available features. If you want to try out a simpler example, change the configuration reference in the [`index.html`](src/index.html) file to `<script src="/assets/basicConfiguration.bundle.js"></script>`. You can edit the `basicConfiguration.bundle.js` file [here](src/luigi-config/basic/basicConfiguration.js).
```
